integration/experiment/ffnet/gen_sweep_to_ffnet.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import argparse
+
+import gen_hdf_from_fsweep as hdf
+import gen_neural_net as nn
+import gen_region_parameters as fmap
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--output',
+                       action='store',
+                       help='Prefix of the output HDF and json file(s)',
+                       default="neural_net_file")
+    parser.add_argument('--description',
+                       help='Description of the neural net, between quotes.',
+                       dest="describe_net",
+                       default="A neural net.")
+    parser.add_argument('--ignore',
+                       help='Comma-separated hashes of any regions to ignore.',
+                       dest="region_ignore",
+                       default=None)
+    parser.add_argument('--frequency_sweep_dirs',
```

Comment:
The convention we're following in the experiment infrastructure scripts (for the most part, at least) is to separate words in options with "-". There are more instances in other places/files.
```suggestion
    parser.add_argument('--frequency-sweep-dirs',
```